DashKit/DashPumpManagerState.swift

```diff
@@ -187,12 +190,23 @@ public struct DashPumpManagerState: RawRepresentable, Equatable {
             self.finishedDoses = []
         }
         
-        if let rawActiveAlerts = rawValue["activeAlerts"] as? PodAlerts.RawValue {
-            self.activeAlerts = PodAlerts(rawValue: rawActiveAlerts)
+        if let rawActiveAlerts = rawValue["activeAlerts"] as? [PumpManagerAlert.RawValue] {
+            self.activeAlerts = []
```

Comment:
[nit] Consider moving `self.activeAlerts = []` before `if` statement and getting rid of `else` clause.

Or consider changing everything in the `if` block to just `self.activeAlerts = rawActiveAlerts.compactMap { PumpManagerAlert(rawValue: $0) }`